### PR TITLE
Feature/file upload2

### PR DESCRIPTION
**Pull request by keatonkraiger**
Added file upload functionality and fixed Startup.cs configuration for localhost communication.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -370,3 +370,8 @@ yarn-error.log*
 *.njsproj
 *.sln
 *.sw?
+
```
Comment (iyves) [inline .gitignore:373]: We should also be excluding the dist directory.

File: PWA/dist/index.html [hunk 2]
```diff
@@ -1 +1 @@
...
.../app.58f89673.js rel=preload as=script><link href=/js/chunk-vendors.6acf834f.js rel=preload as=script><link href=/css/chunk-vendors.7caec713.css rel=stylesheet><link href=/css/app.14e4f460.css rel=stylesheet><link rel=icon type=image/png sizes=32x32 href=/img/icons/favicon-32x32.ico><link rel=icon type=image/png sizes=16x16 href=/img/icons/favicon-32x32.ico><link rel=manifest href=/manifest.json><meta name=theme-color content=#4DBA87><meta name=apple-mobile-web-app-capable content=no><meta name=apple-mobile-web-app-status-bar-style content=default><meta name=apple-mobile-web-app-title content="Vue PWA Rapid Prototype"><link rel=apple-touch-icon href=/img/icons/apple-touch-icon-152x152.png><link rel=mask-icon href=/img/icons/safari-pinned-tab.svg color=#4DBA87><meta name=msapplication-TileImage content=/img/icons/msapplication-icon-144x144.png><meta name=msapplication-TileColor content=#000000></head><body><noscript><strong>We're sorry but Vue-PWA-Rapid-Prototype doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/js/chunk-vendors.6acf834f.js></script><script src=/js/app.58f89673.js></script></body></html>
```
Comment (iyves) [inline PWA/dist/index.html:1]: This directory is the production-ready optimized code, it should be built via an npm command after the repository is cloned, therefore it should not be uploaded to github.

File: PWA/package-lock.json [hunk 3]
```diff
@@ -1008,6 +1008,11 @@
       "integrity": "sha512-shAmDyaQC4H92APFoIaVDHCx5bStIocgvbwQyxPRrbUY20V1EYTbSDchWbuwlMG3V17cprZhA6+78JfB+3DTPw==",
       "dev": true
     },
+    "@popperjs/core": {
```
Comment (iyves) [inline PWA/package-lock.json:1011]: What is popperjs being used for? Is this module used on the code, or should it not be here?

**Comment by ohyonghao** [comment]
Any changes requested please do them on the (move) branch.

**Pull request closed without merging**